File: 283. Move Zeroes/solution.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        cur = 0
        size = len(nums)
        for i in range(0, size):
            if (nums[i] != 0):
                nums[cur] = nums[i]
                cur += 1
        for i in range(cur, size):
            nums[i] = 0

File: 283. Move Zeroes/solution.java
public class Solution {
    public void moveZeroes(int[] nums) {
        int cur = 0;
        int size = nums.length;
        for (int i = 0; i < size; i++) {
            if (nums[i] != 0) {
                nums[cur] = nums[i];
                cur++;
            }
        }
        for(int i = cur; i < size; i++)
            nums[i] = 0;
    }
}
